README.md

```diff
@@ -23,26 +23,26 @@ an update is found, the automation creates a pull request that a human user
 can then review and merge. See [`autoupdate.yaml`](#autoupdateyaml) for
 more information.
 
-### Adding New Images
+### Adding New Artifacts
 
-When adding new images to the repo, please indicate so in the pull request.
+When adding new artifacts to the repo, please indicate so in the pull request.
```

Comment:
It's a little unclear what is meant by "artifact" by itself. We can refer to "artifacts" in most cases, but it is important that we introduce the term as "OCI artifact". This is one example of where I'd use the term "OCI artifacts" (the first usage in this paragraph). Another place I'd use the term "OCI artifact" is in code comments.

I'll let this comment represent all of those cases. There is no clear rule for what should be "artifact" and what should be "OCI artifact", so I'll trust your taste as to which occurrences to modify.